src/main/java/health/medunited/model/LetterRequest.java

```diff
@@ -1,16 +1,19 @@
 package health.medunited.model;
+import java.util.ArrayList;
 
 public class LetterRequest {
     private String contactName;
     private String contactEmail;
     private String contactMessage;
-    private String attachment;
+    private ArrayList<String> attachment;
```

Comment:
I would call it attachments with a final "s", as it is an array

Reply:
will change this later as the other two repositories have this name